e2e/tests/features/journeys/registration/nhsSignupJourneyTest.e2e.ts

```diff
@@ -0,0 +1,237 @@
+      await namePage.assertButtonsVisible();
+      await namePage.assertFormVisible();
+      await namePage.assertCorrectContent();
+      await namePage.assertErrorsHidden();
+    });
+    await test.step("Then I will fill in the last name field and leave the first name field blank", async () => {
+      await namePage.fillLastName("lastName");
+    });
+    await test.step("Then I will press the continue button to move to the next page", async () => {
+      await namePage.clickContinue();
+    });
+    await test.step("I will then see an error message for the first name being empty", async () => {
+      await namePage.assetFirstNameError();
+    });
+    await test.step("I will then go back to fill in the first name field aswell", async () => {
+      await namePage.fillFirstName("firstName");
+    });
+    await test.step("I will then click continue again", async () => {
+      await namePage.clickContinue();
+    });
+    await test.step("I will then see that the error messages are gone if I navigate back to name page", async () => {
+      await namePage.assertErrorsHidden();
+    });
+    await test.step("I will then go back to check that the errors are hidden", async () => {
+      await namePage.clickBack();
+    });
+  });
+
+  test("Both fields empty journey", async ({ namePage }) => {
+    await test.step("Given I have navigated to the name page", async () => {
+      await namePage.goTo();
+    });
+    await test.step("Then I will see the components correctly rendered", async () => {
+      await namePage.assertButtonsVisible();
+      await namePage.assertTextVisible();
+      await namePage.assertFormVisible();
+      await namePage.assertCorrectContent();
+      await namePage.assertErrorsHidden();
+    });
+    await test.step("Then I will press the continue button without filling the input fields", async () => {
+      await namePage.clickContinue();
+    });
+    await test.step("I will then see an error message for both fields being empty", async () => {
+      await namePage.assertBothErrorMessages();
+    });
+    await test.step("I will then go back to fill in both fields", async () => {
+      await namePage.fillFirstName("firstName");
+      await namePage.fillLastName("lastName");
+    });
+    await test.step("I will then click continue again", async () => {
+      await namePage.clickContinue();
+    });
+    await test.step("I will then see that the error messages are gone if I navigate back to name page", async () => {
+      await namePage.assertErrorsHidden();
+    });
+    await test.step("I will then go back to check that the errors are hidden", async () => {
+      await namePage.clickBack();
+    });
+  });
+});
+
+// Unsuccessful DoB page Journeys
+test.describe("Unsuccessful DoB page journeys", () => {
+  test("Empty day and month fields", async ({ namePage, dateOfBirthPage }) => {
+    await test.step("I will correctly navigate through the name page", async () => {
```

Comment:
This step is a little too bulky. 
Try and keep things as segmented and separate as possible around 1-3 methods/actions per step.
For this I'd suggest: 
Given I am on the Name Page (.goTo & assertFormVisible)
And I complete the Form with First and Last Name (Fill & Click)